support-workers/src/test/scala/com/gu/zuora/ZuoraSpec.scala

```diff
@@ -19,6 +20,8 @@ class ZuoraSpec extends AsyncFlatSpec with Matchers {
 
   def uatService: ZuoraService = new ZuoraService(zuoraConfigProvider.get(true), RequestRunners.configurableFutureRunner(30.seconds))
 
+  val earlyDate = OffsetDateTime.of(2010, 1, 1, 0, 0, 0, 0, ZoneOffset.UTC)
```

Comment:
need a long time ago to make sure we pick up the static value with a hard coded date as we are using the real UAT zuora